app/controllers/super_admin/notifications_controller.rb

```diff
@@ -55,6 +55,29 @@ def update
       render :edit
     end
 
+
+    # edit active field displayed in the table
+    def set_active
+
+      notification = Notification.find(params[:id])
+      notification.active = (params[:active] === "1")
+authorize(Notification)
```

Comment:
Please indent the `authorize` line to match the others.
It's mildly pedantic, but it would be good to check the authorization before modifying the record.